src/common/tooltip-utils/tooltip-footer.tsx

```diff
@@ -0,0 +1,33 @@
+import React, { ComponentProps, FC } from 'react'
+import classNames from 'classnames'
+
+type TooltipFooterProps = ComponentProps<'div'> & {
+    type?: string;
+    displayPagination?: boolean;
+    footerIndex?: string;
+}
+
+const TooltipFooter: FC<TooltipFooterProps> = (
+    {
+        displayPagination = false,
+        children,
+        className,
+        footerIndex,
+        type = 'tourtip'
+    }) => {
+    const shouldDisplayPagination = displayPagination && footerIndex !== undefined
+    return (
+        <div className={classNames(`${type}__footer`, className)}>
+            {
+                shouldDisplayPagination &&
```

Comment:
We need to follow marko [version API](https://github.com/eBay/ebayui-core/blob/master/src/components/ebay-tourtip/index.marko#L56). So we will have a EbayTourtipFooter component that accepts a `index` property . It will render the tourtip_index only if index property exists. 
But this base TooltipFooter should just render the `<div className={classNames(`${type}__footer`, className)}>` with its children. The `__index` logic is part of the tourtip instead.